src/main/java/com/epam/aidial/core/service/PublicationService.java

```diff
@@ -505,6 +510,9 @@ private void checkSourceResources(Publication publication) {
     private void checkReviewResources(Publication publication) {
         for (Publication.Resource resource : publication.getResources()) {
             String url = resource.getReviewUrl();
+            if (url == null) {
```

Comment:
You do not need to check it. It is always not null for Publication.PENDING.

Reply:
removed